lib/inc/sai_redis.h

```diff
@@ -43,7 +43,7 @@ void check_notifications_pointers(
 
 // if we don't receive response from syncd in 60 seconds
 // there is something wrong and we should fail
-#define GET_RESPONSE_TIMEOUT (6*60*1000)
```

Comment:
just fyi i remember that this was changed on purpose but i cant remember why ;/

Reply:
Got it. Let us face it again. If we face, I would prefer to fix the other end.